### ISSUE TITLE
Some organization templates not showing up when creating an issue

### ISSUE THREAD

**Issue by elfisher** [opensearch-project/.github#9]
When I go to create an issue in opensearch-project/OpenSearch it looks like the organization templates are not all showing up. Specifically I don't see the proposal template.

![image](https://user-images.githubusercontent.com/10850824/119361140-53ec6c80-bc79-11eb-9af7-aadc9765dd5f.png)

**Comment by dblock** [comment]
I am not sure they are supposed to, are they? I thought you have to actually copy them in the project and .github (this repo) is used as a template to seed new repos. I could be wrong and don't have access to OpenSearch settings, maybe @CEHENKLE can take a look.

**Comment by elfisher** [comment]
I think you're right. When I click the **view organization templates** link at the bottom of creating an issue it opens up:

![image](https://user-images.githubusercontent.com/10850824/119365647-23f39800-bc7e-11eb-9829-4caa4d599146.png)

From what it looks like that is OpenSearch/.github, not opensearch-project/.github. 

Is it safe to assume the opensearch-project/.github repo is acting as a [community health repo](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file)

**Comment by dblock** [comment]
> Is it safe to assume the opensearch-project/.github repo is acting as a [community health repo](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file)?

For sure.

**Comment by dblock** [comment]
https://github.com/opensearch-project/.github/pull/18

**Comment by dblock** [comment]
We've updated templates across all OpenSearch plugins in #64. It was pretty tedious. Next I think we need some kind of automation to keep these templates up-to-date across all repos, maybe an app or a bot?

**Comment by elfisher** [comment]
> We've updated templates across all OpenSearch plugins in #64. It was pretty tedious. Next I think we need some kind of automation to keep these templates up-to-date across all repos, maybe an app or a bot?

Agreed. We will have future changes proposed to these templates. For example, I created [this](https://github.com/opensearch-project/.github/pull/71) today.

Automation to publish these across the repos will be very useful.
